tox.ini

```diff
@@ -18,7 +18,9 @@ commands =
     pip check
     {toxinidir}/lints/queuelint.py
     flake8 --show-source
-    pre-commit run --all-files
+    black --check
+    shellcheck initialize_data.sh
+    shellcheck docker/entrypoint.sh
```

Comment:
here we replace `pre-commit` with the python commands.  `shellcheck` does not appear to accept wildcards